tests/find/index.html

```diff
@@ -39,6 +39,7 @@
     <script src='./test-suite-1/test.exists.js'></script>
     <script src='./test-suite-1/test.fields.js'></script>
     <script src='./test-suite-1/test.issue66.js'></script>
+    <script src='./test-suite-1/test.issue9002f.js'></script>
```

Comment:
Why `9002f` instead of `9002`?

Reply:
Because it tests 9002 and 9003. :laughing: I admit its a bit weird.

Reply:
:laughing:  is `f` the plural of `2`?

Reply:
I stole it from citations. There you sometimes say `see page 50f` to say its on page 50 and page 51.

Reply:
TIL :upside_down_face: 

Would `'./test-suite-1/test.issue9002-9003.js'` be more obvious?

Also sorry for the superficial review - I'll try to look at the actual details tomorrow.

Reply:
Don’t worry. Sure, can do.